pom.xml

```diff
@@ -63,6 +63,10 @@
       <groupId>io.netty</groupId>
       <artifactId>netty-codec-mqtt</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-codec-haproxy</artifactId>
```

Comment:
can you set this dependency to optional ?

Reply:
Do you mean `test` scope?

Reply:
test scope would be fine too.

Vert.x Core defines an optional dependency onto this artifact because it needs to compile with it.